pkg/agent/baker.go

```diff
@@ -48,11 +51,11 @@ func (t *TemplateGenerator) getLinuxNodeCustomDataJSONObject(config *datamodel.N
 		panic(e)
 	}
 
-	return fmt.Sprintf("{\"customData\": \"[base64(concat('%s'))]\"}", str)
+	return fmt.Sprintf("{\"customData\": \"%s\"}", str)
```

Comment:
alternatively, we can do base64.StdEncoding.EncodeToString([]byte(str)) here and save the change from caller side.